tests/main/interfaces-dbus/task.yaml

```diff
@@ -68,6 +68,13 @@ execute: |
     echo "And plug is disconnected by default"
     snap interfaces | MATCH "$DISCONNECTED_PATTERN"
 
+    # FIXME This is a pretty bad check for unsupported confinement
```

Comment:
I would like to run the first part of this as it exercise the dbus service setup through the configuration in the snap.yaml. snapd still puts the dbus configuration file in place to allow ownership over bus names which will be still validated if we have snap confinement or not.